resources/language/Croatian/strings.po

```diff
@@ -0,0 +1,270 @@
+# Croatian translation for Kodi Media Center Netfilx addon
+# Addon Name: Netflix
+# Addon id: plugin.video.netflix
+# Addon Provider: libdev + jojo + asciidisco
+msgid ""
+msgstr ""
+"Project-Id-Version: XBMC-Addons\n"
+"Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
+"POT-Creation-Date: 2017-01-01 12:00+0000\n"
+"PO-Revision-Date: 2017-10-13 20:02+0200\n"
+"Last-Translator: gogo\n"
+"Language-Team: Croatian\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: hr\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.0.4\n"
+
+msgctxt "Addon Summary"
+msgid "Netflix"
+msgstr "Netflix"
```

Comment:
Can you remove that `msgstr` contents, if `msgid` & `msgstr` are the same, `msgstr` should be empty